### ISSUE TITLE
Feature: Quality Picker

### ISSUE THREAD

**Issue by Baa14453** [GRVYDEV/Project-Lightspeed#8]
A QoL thing we're all used to is automatic and or manual quality selection.
At the expense of encoding additional video feeds a selection menu that lets the user choose which quality they want to use eg. 480, 720, 1080p etc etc.

**Comment by GRVYDEV** [comment]
So I will have to dig into this more but this could actually ruin this project and make it VERY difficult for someone to self host due to cpu constraints. Again I will have to look into it more but that is my initial concern.

**Comment by Daniele122898** [comment]
I mean you can make it optional on the hoster side. Such that the hoster can specify in some config or similar if he wants to allow for quality selection or not :)

**Comment by GRVYDEV** [comment]
> I mean you can make it optional on the hoster side. Such that the hoster can specify in some config or similar if he wants to allow for quality selection or not :)

Yeah we could do that however due to the trans-coding required this would make lightspeed no longer sub second. It is definitely something I want to do in the future but as of right now it is a large undertaking
